hudi-common/src/main/java/org/apache/hudi/common/table/timeline/versioning/v2/CompletionTimeQueryViewV2.java

```diff
@@ -102,7 +106,14 @@ public CompletionTimeQueryViewV2(HoodieTableMetaClient metaClient) {
   public CompletionTimeQueryViewV2(HoodieTableMetaClient metaClient, String eagerLoadInstant) {
     this.metaClient = metaClient;
     this.instantTimeToCompletionTimeMap = new ConcurrentHashMap<>();
-    this.cursorInstant = InstantComparison.minInstant(eagerLoadInstant, metaClient.getActiveTimeline().firstInstant().map(HoodieInstant::requestedTime).orElse(""));
+    this.cursorInstant = metaClient.getActiveTimeline().firstInstant().map(HoodieInstant::requestedTime).orElse("");
```

Comment:
Hi, the `eagerLoadInstant` is not configurable now, wondering how this patch could improve the performance?
can we by default set the eagerLoadInstant as the first active instant and not load loading the archived timeline instead? Not a fan of these flags variables introduced.